website/docs/d/agent_pool.html.markdown

```diff
@@ -9,9 +9,7 @@ description: |-
 
 Use this data source to get information about an agent pool.
 
-~> **NOTE:** This data source requires using the provider with Terraform Cloud and a Terraform Cloud 
-for Business account. 
-[Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing).
+~> **NOTE:** This resource requires using the provider with [Terraform Cloud](https://www.hashicorp.com/products/terraform/pricing).
```

Comment:
Arguably, all the resources require Terraform Cloud... One bit of lost information here is that agent pools require a Cloud Business plan. Is your intent to improve the link text (by omitting the word "here", for instance)? I might suggest

```suggestion
~> **NOTE:** This resource requires using the provider with a [Terraform Cloud Business(https://www.hashicorp.com/products/terraform/pricing) account.
```